indexer/osm_editor.cpp

```diff
@@ -0,0 +1,109 @@
+#include "osm_editor.hpp"
+
+#include "indexer/feature_decl.hpp"
+#include "indexer/index.hpp"
+
+#include "std/map.hpp"
+#include "std/set.hpp"
+
+namespace osm
```

Comment:
Our current approach is an injection of osm code into an indexer. I don't see any reason to put Editor class into indexer namespace, as it is not edit any `index` (which is an incorrect word by itself for the library), but edits osm features.

Don't want to waste time arguing about google standard, which has different statements about 'projects' and 'nested namespaces'.
osm::Editor is the correct name for the class, as it works with OSM data (it's Impl is empty now, but upcoming commits fill it up). May be it will be moved to a separate osm library, if we really need it. And osm::Editor will be a correct name in that case too.